.eslintrc.js

```diff
@@ -22,7 +22,11 @@ module.exports = {
   plugins: ['react', '@typescript-eslint', '@emotion', 'cypress'],
   rules: {
     // note you must disable the base rule as it can report incorrect errors
-    'no-unused-vars': 'off',
+    'no-undef': 'warn',
```

Comment:
I reactivated Eslint now and add this here to at least get warnings about missing imports. Before we add this with `error` we need to fix quite some stuff. We should enable much more lint checks and fix the code as not using this is like driving a car without using a seat belt.